common/test/acceptance/pages/studio/settings.py

```diff
@@ -234,3 +238,50 @@ def refresh_page(self, wait_for_confirmation=True):
             ).fulfill()
         self.wait_for_require_js()
         self.wait_for_ajax()
+
+    @staticmethod
+    def get_asset_path(file_name):
```

Comment:
This code also exists in /edx-platform/common/test/acceptance/tests/discussion/test_cohort_management.py. Don't duplicate it-- find a place to share the code.